### PR TITLE
Fixed compatibility with breitbandmessung.de

### PR DESCRIPTION
**Pull request by kosekmi**

### CODE REVIEW

File: src/chrome/content/rules/Breitbandmessung.de.xml [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 	  drsf.breitbandmessung.de
 	  drsfrtt.breitbandmessung.de
 	  drsfv4.breitbandmessung.de
-
 	Timeout:
 	  drsm.breitbandmessung.de
 -->
 <ruleset name="Breitbandmessung.de">
 
+	<exclusion pattern="^http://breitbandmessung.de/(?:test|images|api|public)/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Breitbandmessung.de.xml]: Please remove the non-capturing group.

**Comment by J0WI** [comment]
Can you explain why this exclusion is required?
Please also add a comment about the reason in the ruleset itself.

**Comment by J0WI** [comment]
Now you replaced every tab with whitespaces

**Comment by kosekmi** [comment]
Sorry guys, I don't know wth is going on with my editor. Whitespaces should now be replaced with tabs.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**